### PR TITLE
[Breq 1736] VMs cpu utilization script

### PR DESCRIPTION
**Pull request by waqasnazir03**
This PR addresses #1736.

### CODE REVIEW

File: voithos/lib/util/util.py [hunk 1]
```diff
@@ -4,16 +4,45 @@
 import pkg_resources
 import sys
 import subprocess
+import psutil
+import re
 import voithos.lib.config as config
 import voithos.lib.aws.ecr as ecr
 import voithos.lib.aws.s3 as s3
+import datetime
```
Comment (kylep) [inline voithos/lib/util/util.py:12]: This is confusing, why are you importing datetime and also date from datetime below it?

File: voithos/lib/util/util.py [hunk 2]
```diff
@@ -4,16 +4,45 @@
...
+def get_instances_cpu_usage():
+    """ Displays cpu utilization of all the server on host"""
+    for proc in psutil.process_iter():
+        now = datetime.datetime.now()
+        timestamp = now.strftime("%Y-%m-%d-%H-%M-%S")
+        if proc.name() == 'qemu-system-x86_64':
+           pid = proc.pid
+           cpu_percent = proc.cpu_percent(interval=0.5)
+           cmd_info = proc.cmdline()
+           uuid = ([e for e in cmd_info if "uuid=" in e])
```
Comment (kylep) [inline voithos/lib/util/util.py:30]: Why the brackets?

File: voithos/lib/util/util.py [hunk 3]
```diff
@@ -4,16 +4,45 @@
...
 from voithos.constants import KOLLA_IMAGE_REPOS, OFFLINE_DEPLOYMENT_SERVER_PACKAGES
 from voithos.lib.system import error, shell
 
 
+def get_instances_cpu_usage():
+    """ Displays cpu utilization of all the server on host"""
+    for proc in psutil.process_iter():
+        now = datetime.datetime.now()
+        timestamp = now.strftime("%Y-%m-%d-%H-%M-%S")
+        if proc.name() == 'qemu-system-x86_64':
```
Comment (kylep) [inline voithos/lib/util/util.py:26]: When the whole loop needs to go inside an if check like this, consider writing it as:
```
for proc in psutil.process_iter():
    if proc.name() != 'qemu-system-x86_64':
        continue  #  Skip this process, we only want qemu-system VMs
    ....
```

**Review by kylep — CHANGES_REQUESTED** [review]

**Comment by waqasnazir03** [comment]
@kylep 
I have addressed your comments. Please have another look when you get a chance.

**Review by kylep — APPROVED** [review]

**Pull request merged**